src/components/EditProject/EditProject.md

```diff
@@ -0,0 +1,41 @@
+### Edit Project Page
+
+This component will help to Edit the created Project
+
+```js
+import EditProject from './EditProject';
+import Button from '@material-ui/core/Button';
+import Grid from '@material-ui/core/Grid';
+const [projectData, setProjectData] = React.useState();
+
+import { API } from '../../store/api';
+
+const loadText = () => {
+  API.get('autographa/projects?project_name=Operation%20Agape')
```

Comment:
Operation%20Agape should Ideally not be hardcoded, as it may not always  exist in the demo server and this example will not work without it. It should Ideally show a project dropdown and select a project from there. For now its fine we can take it up later during refactoring